internal/pkg/describe/env.go

```diff
@@ -131,6 +133,32 @@ func (d *EnvDescriber) Describe() (*EnvDescription, error) {
 	return d.description, nil
 }
 
+// Manifest returns the contents of the manifest used to deploy an environment stack.
+func (d *EnvDescriber) Manifest() ([]byte, error) {
+	tpl, err := d.cfn.StackMetadata()
+	if err != nil {
+		return nil, err
+	}
+
+	metadata := struct {
+		Manifest string `yaml:"Manifest"`
```

Comment:
Why do we capitalize `manifest`?

Reply:
Otherwise, `yaml.Unmarshal` won't be able to populate the field. The types can be private but the fields must be public

Reply:
Oh i meant `yaml:"Manifest"`

Reply:
That's how we write it in the CFN template:
```
Metadata:
  Version:
  Manifest:
```

Reply:
I see it looks good since all the other fields are capitalized as well.